src/main/java/jenkins/plugins/slack/workflow/SlackSendStep.java

```diff
@@ -300,13 +313,12 @@ protected SlackResponse run() throws Exception {
 
         JSONArray getAttachmentsAsJSONArray() throws Exception {
             final TaskListener listener = getContext().get(TaskListener.class);
-        	final String jsonString;
-        	if (step.attachments instanceof String) {
-        		jsonString = (String) step.attachments;
-        	}
-        	else {
-        		jsonString = JsonOutput.toJson(step.attachments);
-        	}
+            final String jsonString;
```

Comment:
what changed in this block? tabs to spaces or something? can't tell from github

Reply:
it seems something got merged into master that had a mix of tabs and spaces and my IDE "fixed" it...